<launch>
	<!-- turtle's moving pattern: 'forward_turn' -->
	<arg name="moving_pattern" default="forward_turn"/>

	<node name="move_turtle" pkg="ninja_turtles" type="move_turtle_node">
		<param name="moving_pattern" type="string" value="$(arg moving_pattern)"/>
	</node>

	<node name="turtlesim_node" pkg="turtlesim" type="turtlesim_node"/>
</launch>
